fetch_maps/package.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<package>
+  <name>fetch_maps</name>
+  <version>0.0.1</version>
+  <description>The fetch_maps package</description>
+
+  <maintainer email="ablasdel@fetchrobotics.com">Aaron Blasdel</maintainer>
+  <author>Michael Gregg</author>
+  <author>Michael Ferguson</author>
+  <license>BSD</license>
+
+  <buildtool_depend>catkin</buildtool_depend>
+  <run_depend>slam_karto</run_depend>
```

Comment:
We want to keep using the launch file so that the parameters/topics are correctly setup. fetch_navigation is the best place for this launch file. Also, fetch_navigation becomes where we inject the slam_karto depend so that it ends up on the robot and available in a debian environment being used for gazebo